core/domain/skill_services.py

```diff
@@ -262,6 +264,21 @@ def get_skill_descriptions_by_ids(skill_ids):
         if skill_summary_model is not None:
             skill_id_to_description_dict[skill_summary_model.id] = (
                 skill_summary_model.description)
+
+    deleted_skill_ids = []
+    for skill_id in skill_ids:
+        if skill_id not in skill_id_to_description_dict:
+            skill_id_to_description_dict[skill_id] = None
+            deleted_skill_ids.append(skill_id)
+
+    if deleted_skill_ids:
+        deleted_skills_string = ', '.join(deleted_skill_ids)
+        if feconf.CAN_SEND_EMAILS:
```

Comment:
Add logging.error(...) above this line as well.

Reply:
Done.